omegaconf/_utils.py

```diff
@@ -334,6 +334,23 @@ class ValueKind(Enum):
     INTERPOLATION = 2
 
 
+def _is_missing_value(value: Any) -> bool:
+    from omegaconf import Node
+
+    if isinstance(value, Node):
+        value = value._value()
+    return _is_missing_literal(value)
+
+
+def _is_missing_literal(value: Any) -> bool:
+    from omegaconf import MISSING, Node
+
+    assert not isinstance(value, Node)
+    ret = isinstance(value, str) and value == MISSING
+    assert isinstance(ret, bool)
+    return ret
+
+
```

Comment:
Because I had rebased on top of your `missing` branch that hadn't been merged yet (see https://github.com/omry/omegaconf/pull/545#discussion_r583249743)

I did this because I needed these changes to properly address some of your comments.

Anyway, I just pushed a new version rebased on top of master, this should get rid of it.

Reply:
Resolving since this is not part of this PR anymore